### DISCUSSION TITLE
Multipart upload bug. Where extractFiles function in latest 3.2.2 release?

### DISCUSSION THREAD

**Discussion by max107** [urql-graphql/urql#3070] [Q&A]
### Describe the bug

Package @urql/exchange-multipart-fetch [marked as deprecated](https://formidable.com/open-source/urql/docs/api/multipart-fetch-exchange/). 

Latest release should support multipart upload as described here

> and @urql/core now supports GraphQL Multipart Requests natively

but i have no see ![extractFiles](https://github.com/urql-graphql/urql/blob/8a5de33f449824821b4ee6510b610938d03e4b09/packages/core/src/utils/variables.ts#L82) function and multipart uploads not working as expected.

```
rg 'extractFiles' . | wc -l
       0
```

installed package

```
"@urql/core@>=3.0.0", "@urql/core@>=3.1.1", "@urql/core@>=3.2.0", "@urql/core@>=3.2.2":
  version "3.2.2"
  resolved "https://registry.yarnpkg.com/@urql/core/-/core-3.2.2.tgz#2a44015b536d72981822f715c96393d8e0ddc576"
  integrity sha512-i046Cz8cZ4xIzGMTyHZrbdgzcFMcKD7+yhCAH5FwWBRjcKrc+RjEOuR9X5AMuBvr8c6IAaE92xAqa4wmlGfWTQ==
  dependencies:
    wonka "^6.1.2"
```

### Reproduction

-

### Urql version

urql@3.2.2

### Validations

- [X] I can confirm that this is a bug report, and not a feature request, RFC, question, or discussion, for which [GitHub Discussions](https://github.com/urql-graphql/urql/discussions) should be used
- [X] Read the [docs](https://formidable.com/open-source/urql/docs/).
- [X] Follow our [Code of Conduct](https://github.com/urql-graphql/urql/blob/main/CODE_OF_CONDUCT.md)

**Comment by JoviDeCroock** [comment]
This is unreleased on stable 😅 these are changes lined up for the next major, this is also not a bug so converting to a discussion

**Reply by max107** [reply]
haha, sorry. my bad.
